File: go/1035.go
package main

import "fmt"

func main() {
	var a, b, c, d int
	fmt.Scanf("%d %d %d %d", &a, &b, &c, &d)
	var aux int = 0
	if d > a && b > c {
		if d+c > b+a {
			if c > 0 && d > 0 {
				if a%2 == 0 {
					aux = 1
				}
			}
		}
	}
	if aux == 0 {
		fmt.Println("Valores não aceitos")
	} else {
		fmt.Println("Valores aceitos")
	}
}

File: C/src/1035.c
#include <stdio.h>

int main() {
  int a, b, c, d, aux = 0;
  scanf("%i %i %i %i", &a, &b, &c, &d);
  if (d > a && b > c)
    if (d + c > b + a)
      if (c > 0 && d > 0)
        if (a % 2 == 0)
          aux = 1;

  if (aux == 0)
    printf("Valores não aceitos\n");
  else
    printf("Valores aceitos\n");
  return 0;
}